cluster/images/kubelet-checker/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine
```

Comment:
Should this reference a specific version? Otherwise it seems like it won't be possible to re-create an identical image (e.g. from a release tag).